topdown/crypto.go

```diff
@@ -122,6 +123,137 @@ func builtinCryptoX509ParseAndVerifyCertificates(_ BuiltinContext, operands []*a
 	return iter(valid)
 }
 
+type verifyOptions struct {
```

Comment:
question: is there a reason we can't use the golang type? https://pkg.go.dev/crypto/x509#VerifyOptions